lib/is-arguments.js

```diff
@@ -2,34 +2,31 @@
 
 var getClass = require("./get-class");
 
+// This is for IE11, which can't reliably detect arguments
+// However, it allows use of callee in strict mode without blowing up
+var engineCanDetectArguments = (function test() {
+    return getClass(arguments);
+})();
+
 /**
- * Returns `true` when `object` is an `arguments` object, `false` otherwise
```

Comment:
Your editor is quite aggressive with the JSDoc, and doesn't understand the settings file :(

Reply:
thanks, for spotting this, it now fixed